<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>925</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="locateselect_combobox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>850</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="makescedule_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>850</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="where_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>860</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Locate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotologin_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Login</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>801</width>
     <height>821</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>09:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>09:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>19:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>19:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>21:00</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>21:30</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>22:00</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>S</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>M</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>T</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>W</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>T</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>F</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>S</string>
    </property>
   </column>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>locateselect_combobox</zorder>
  <zorder>makescedule_button</zorder>
  <zorder>where_label</zorder>
  <zorder>gotologin_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
